### PR TITLE
e2e fix: 'my-company/units.e2e'

### PR DESCRIPTION
**Pull request by kpawelczak**

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/my-company/units.e2e.cy.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe('A11y - Units List Keyboard Controls', () => {
     cy.focused().type('{downArrow}');
     cy.focused().should('have.id', 'Rustic Services');
     cy.focused().type('{downArrow}');
-    cy.focused().should('have.id', 'Rustic Services');
+    cy.focused().should('have.id', 'unit-kris1');
```
Comment (pawelfras) [inline projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/my-company/units.e2e.cy.ts]: `unit-kris1` value has been added by @Platonn when testing Organization feature, but as far as I remember, only to our dev environment. Please verify if test fails against CI environment

File: projects/storefrontapp/src/environments/environment.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ export const environment: Environment = {
   occBaseUrl: buildProcess.env.CX_BASE_URL,
   occApiPrefix: '/occ/v2/',
   cds: buildProcess.env.CX_CDS ?? false,
-  b2b: buildProcess.env.CX_B2B ?? false,
+  b2b: true,
```
Comment (pawelfras) [inline projects/storefrontapp/src/environments/environment.ts]: heads-up: revert the change before merge

File: projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/my-company/units.e2e.cy.ts (hunk 1 above)
Comment (rmch91) [reply projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/my-company/units.e2e.cy.ts]: Verified on CI, and this part works. There are no units from Kris :)

**Review by rmch91 — APPROVED** [review]
LGTM
